src/components/implementation/tests/micro_booter/mb_tests.c

```diff
@@ -332,69 +157,125 @@ spinner(void *d)
 		;
 }
 
+void
+clear_sched(int* rcvd, thdid_t* tid, int* blocked, cycles_t* cycles, tcap_time_t* thd_timeout)
+{
+	while (cos_sched_rcv(BOOT_CAPTBL_SELF_INITRCV_CPU_BASE, RCV_ALL_PENDING, 0,
+			     rcvd, tid, blocked, cycles, thd_timeout) != 0)
+		;
+}
+
+#define NUM_TEST 16 			/* Iterator NUM */
+#define MAX_THRS 4  			/* Max Threshold Multiplier */
```

Comment:
We've been using `THD` as short for `THREAD` everywhere in the code. `THR` is new. ;-)

Reply:
FIXED